build/cmake/init.cmake

```diff
@@ -348,6 +348,14 @@ if(wxUSE_LIBLZMA)
     endif()
 endif()
 
+if(wxUSE_LIBWEBP)
+    find_package(WebP) # TODO: select specific components as soon as upstream-supplied WebPConfig.cmake supports it
```

Comment:
Very minor question, and probably more to @MaartenBent, but do we want to use mixed-case for this one or `WEBP` for consistency with all the other 3rd party libraries?

Reply:
This depends on the package you want to find. It should match the name of the `FindLibraryName.cmake` or `LibraryNameConfig.cmake` file.

For example, the CMake provided modules are `FindJPEG.cmake` and `FindPNG.cmake` and `FindOpenGL.cmake`, so it should match that name.
I think `Webkit` provides `FindWebP.cmake` so the current package name is correct.

If it does not match, it will show a warning:
```
  The package name passed to `find_package_handle_standard_args` (OpenGL)
  does not match the name of the calling package (OPENGL).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
```

Reply:
Sorry, I was somehow under (wrong) impression that this matching was case-insensitive, but if it's case-sensitive then there is no question about what to use here, sorry again for the misunderstanding.